Makefile

```diff
@@ -36,6 +36,7 @@ endtoend-test: docker-build
 	docker build -t $(IMAGE)-dirty .
 	cd e2e && \
 	docker build -t c.txtdirect.org/tester:dirty . && \
+ 	docker run -d -p 5000:5000 --name registry registry:2 && \
```

Comment:
Can we pin this to a specific docker registry version?

Reply:
The currently available tags are:
- 2.7.1, 2.7, 2, latest
- 2.6.2, 2.6

Since "2.7.1, 2.7, 2" are all the same image, is it needed to switch the tag?

Reply:
Question is how much we trust Docker not to break during minor versions.
We could be specific and use 2.7.1, but happy to keep it as is. Up to you.

Reply:
Yeah I agree that Docker may mess up something during minor versions.
Will change it to `2.7.1`.